main.py

```diff
@@ -1,15 +1,18 @@
-import pygame
-from pygame.locals import *
-import os
-import sys
+import json
 import math
+import os
+import pygame
 import random
+import sys
 import time
-import json
 
-class Player(object):
-	
-	imgs = [pygame.image.load(os.path.join('pics', "player-"+ str(x) + '.png')) for x in range(1,10)]
+
+# The class Player descibes the the player object. It has info such as the x positon, y position, width, height, contour coordinates and
+# a draw method to draw the player's images onto the screen hence creating the animation effect.
+class Player:
+	# Loading player images
+	num_of_player_imgs = 9

```

Comment:
Either use camel case (numOfPlayers) or snake case (num_of_players) and not both. Maintain any one style throughout your program.